.github/workflows/main.yml

```diff
@@ -61,6 +61,9 @@ jobs:
       with:
         path: ~/.gradle/wrapper
         key: ${{ runner.os }}-gradle-wrapper-${{ hashFiles('gradle/wrapper/gradle-wrapper.properties') }}
+    # Run ktlint
+    - name: lint
+      run: ./gradlew ktlint
     # Run ksp generated tests
```

Comment:
@neetopia I added a lint check to the workflow. I am not that familiar with .github workflows so I am unsure if this is correct or not. I also suspect it might run in all 3 environments (any tips)?
Wanted to add this check in (even if it was wrong) to prevent regressions, I believe there might already be regressions since I did a rebase.

Reply:
doesn't look to be an issue